python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -312,8 +312,8 @@ def _convolve(self, exposure: afwImage.Exposure, modelPsf: afwDetection.Gaussian
         # side, which is region of interest. The PSF matching sets NO_DATA mask
         # bit in the outermost k pixels. To account for these nans in the
         # edges, the subExposure needs to be expanded by another k pixels.
-        # So grow the bounding box initially by k pixels on either side.
-        pixToGrow = self.config.modelPsfMatch.kernel.active.kernelSize//2
+        # So grow the bounding box initially by 2k pixels on either side.
```

Comment:
I just thought that this k meant thousand. It is more obvious if you read the whole comment though and not just the changes.

Reply:
k for thousand is not generally capitalised.